projects/iot-cloud/stack.yml

```diff
@@ -118,7 +118,16 @@ services:
      - broker
      - influxdb
 
-
+  lora2mqtt:
+    image: beia/lora2mqtt
+    environment:
+      MQTT_HOST: 'broker'
+      VIRTUAL_HOST: 'lora.beia188.dfilip.xyz'
+      VIRTUAL_PORT: 80
+      LETSENCRYPT_HOST: 'lora.beia188.dfilip.xyz'
```

Comment:
Why do you need to expose this?

Reply:
Huh? How else you should expose a HTTPS callback-server?

Reply:
This is how Lora works? Ok, in that case, please use a beia domain.

Reply:
That is the best integration option with TTN. Changing the domain is on the TODO list.

Reply:
We are pending for lora-aggregator.beia-temetrie.ro